jenkins/common.groovy

```diff
@@ -168,7 +168,7 @@ def checkForBrokenLinks(String buildLogPath, String envName) {
 
     // Until slackUploadFile works...
     def linkCount = sh(returnStdout: true, script: "cd /application && wc -l ${csvFileName} | cut -d ' ' -f1") as Integer
-    slackSend message: "${linkCount - 1} broken links found in the ${envName} build on `${env.BRANCH_NAME}`\n${env.RUN_DISPLAY_URL}".stripMargin(),
```

Comment:
The `- 1` was to account for the header line. Turns out `wc -l` counts the newline characters, though, so the count is actually off by one. There's one more line than newline characters, so counting the newline characters effectively ignores the header line.